docs/inventory/settings.md

```diff
@@ -50,8 +50,14 @@ Checking this will allow the usage of Batches. Items will have a Has Batch
 option if the item is being tracked. It will also enable the Batch entry type
 which can be navigated to by using the Quick Search.
 
+### Enable Serial Number
+
+Checking this will allow the usage of Serial Number. Items will have a Has Serial No
+option if the item is being tracked. It will also enable the Serial No entry type
+in the Inventory tab.
```

Comment:
What do you mean by "Serial No entry type in the Inventory tab"?

Reply:
in [#607 ](https://github.com/frappe/books/pull/605), a page was added for Serial Numbers in the Inventory section of the Sidebar.  [It is removed in](https://github.com/frappe/books/pull/605#discussion_r1183309116)  [#617](https://github.com/frappe/books/pull/617)

will remove that sentence.